docs/getting-started/integration.md

```diff
@@ -15,87 +15,82 @@ Chart.js can be integrated with plain JavaScript or with different module loader
-  BarElement,
-  PointElement,
-  BarController,
-  BubbleController,
-  DoughnutController,
-  LineController,
-  PieController,
-  PolarAreaController,
-  RadarController,
-  ScatterController,
-  CategoryScale,
-  LinearScale,
-  LogarithmicScale,
-  RadialLinearScale,
-  TimeScale,
-  TimeSeriesScale,
-  Decimation,
-  Filler,
-  Legend,
-  Title,
-  Tooltip,
-  SubTitle
-} from 'chart.js';
-
-Chart.register(
-  ArcElement,
-  LineElement,
-  BarElement,
-  PointElement,
-  BarController,
-  BubbleController,
-  DoughnutController,
-  LineController,
-  PieController,
-  PolarAreaController,
-  RadarController,
-  ScatterController,
-  CategoryScale,
-  LinearScale,
-  LogarithmicScale,
-  RadialLinearScale,
-  TimeScale,
-  TimeSeriesScale,
-  Decimation,
-  Filler,
-  Legend,
-  Title,
-  Tooltip,
-  SubTitle
-);
-
-const myChart = new Chart(ctx, {...});
+import Chart from 'chart.js/auto';
 ```
 
-A short registration format is also available to quickly register everything.
+Or if you are planning to optimize the bundle later, you can start by importing and registering `registerables`, which also ensures all features are available:
 
 ```javascript
 import { Chart, registerables } from 'chart.js';
 Chart.register(...registerables);
 ```
 
-And finally there is a separate path to do just the above for you, in one line:
-
-```javascript
-import Chart from 'chart.js/auto';
-```
-
-## CommonJS
-
-Because Chart.js is an ESM library, in CommonJS modules you should use a dynamic `import`:
-
-```javascript
-const { Chart } = await import('chart.js');
-```
+### Bundle optimization
+
+When optimizing the bundle, instead of the `registerables`, you need to import and register the components that are needed in your application.
+
+The options are categorised (inspect `registerables`) into controllers, elements, plugins, scales. You can pick and choose many of these, e.g. if you are not going to use tooltips, don't import and register the `Tooltip` plugin. But each type of chart has its own bare-minimum requirements (typically the type's controller, element(s) used by that controller and scale(s)):
+
+* Bar chart
+  * `BarController`
+  * `BarElement`
+  * Default scales: x: `CategoryScale`, y: `LinearScale`
+* Bubble chart
+  * `BubbleController`
+  * `PointElement`
+  * Default scales: x/y: `LinearScale`
+* Doughnut chart
+  * `DoughnutController`
+  * `ArcElement`
+  * Not using scales
+* Line chart
+  * `LineController`
+  * `LineElement`
+  * `PointElement`
+  * Default scales: x: `CategoryScale`, y: `LinearScale`
+* Pie chart
+  * `PieController`
+  * `ArcElement`
+  * Not using scales
+* PolarArea chart
+  * `PolarAreaController`
+  * `ArcElement`
+  * Default scale: r: `RadialLinearScale`
+* Radar chart
+  * `RadarController`
+  * `LineElement`
+  * `PointElement`
+  * Default scale: r: `RadialLinearScale`
+* Scatter chart
+  * `ScatterController`
+  * `PointElement`
+  * Default scales: x/y: `LinearScale`
+
+The plugin imports are:
```

Comment:
are some of these required and some optional?
```suggestion
The available plugins are:
```

Reply:
All plugin options are optional